engine/src/test/java/io/camunda/zeebe/engine/processing/job/JobUpdateRetriesTest.java

```diff
@@ -158,4 +160,28 @@ public void shouldRejectUpdateRetriesIfRetriesLessThanZero() {
     // then
     Assertions.assertThat(jobRecord).hasRejectionType(RejectionType.INVALID_ARGUMENT);
   }
+
+  @Test
+  public void shouldMakeJobActivatableAfterIncreasingRetries() {
```

Comment:
:question: Why do we change the behavior of updating job retries?

Before, the job update command doesn't make the job activatable. Instead, only the incident resolve command made the job activable again.

Reply:
Doesn't it make sense that if a job has retries, it is possible to activate it? Do we always have to use this command in combination with an incident resolve?

Before we still have this bug where the job stays activatable. So increasing the retries would mean the job is activatable like before. I suppose this could still lead to the consistency issue if the incident is not resolve so perhaps it's better to leave this out.